modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/TreeFilter.js

```diff
@@ -26,6 +27,9 @@ import {
 	visit,
 } from '../utils/tree-utils';
 
+const SEARCH_QUERY_MIN_LENGHT = 2;
+const SEARCH_INPUT_DEBOUNCE = 500;
```

Comment:
This value is a little bit high and makes the filtering look slow. I think that 200 would be a better value, don't you think?

Reply:
Totally agree. Thxs